tests/polars/test_polars_components.py

```diff
@@ -194,6 +196,47 @@ def test_check_dtype(data, from_dtype, check_dtype):
         )
 
 
+def test_check_data_container():
+    @polars_engine.Engine.register_dtype
+    class MyTestStartsWithID(polars_engine.String):
+        """
+        Test DataType which expects strings starting with "id_"
+        """
+
+        def check(
+            self,
+            pandera_dtype: DataType,
+            data_container: Optional[polars_engine.PolarsDataContainer] = None,
+        ) -> Union[bool, Iterable[bool]]:
+            if key := data_container.key:
```

Comment:
`key` is really only used once in the function body: I'd recommend just using `data_container.key`

Reply:
Updated